change/@fluentui-react-native-tokens-2020-05-13-14-36-09-sammy-finalAppium.json

```diff
@@ -0,0 +1,8 @@
+{
+  "type": "patch",
+  "comment": "Merge branch 'master' of https://github.com/microsoft/fluentui-react-native into sammy/finalAppium",
```

Comment:
beachball might have done something weird, but I don't think you should need changes for merging master in? Maybe the merge was a little messy?

Reply:
I don't know much about beach ball, but I'll look into it. Nothing was crazy about the merge

Reply:
Should I delete the changes associated with the merge?

Reply:
I may have missed it, but I didn't see any changes in your source to files under those packages. In that case, yeah just delete them

Reply:
Deleted in newest commit